tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{36,37,38,39,py3}-dj{22,30,31,32}-{unittest,pytest}
-    py{38,39}-djmain-{unittest,pytest}
+    py{36,37,38,39,310,py3}-dj{22,30,31,32}-{unittest,pytest}
```

Comment:
Django versions 2.2 through to 3.1 will not support Python 3.10. (Django 3.2 will in it's next release in early November)